store/store.go

```diff
@@ -1191,6 +1212,116 @@ func (s *Store) Find(ctx context.Context, search *Search, user *auth.UserState)
 		return nil, ErrBadQuery
 	}
 
+	q := url.Values{}
+	q.Set("fields", strings.Join(s.findFields, ","))
+	q.Set("architecture", s.architecture)
+
+	if search.Private {
+		q.Set("private", "true")
+	}
+
+	if search.Prefix {
+		q.Set("name", searchTerm)
+	} else {
+		if search.CommonID != "" {
+			q.Set("common-id", search.CommonID)
+		}
+		if searchTerm != "" {
+			q.Set("q", searchTerm)
+		}
+	}
+
+	// section is "category" in search v2
+	if search.Section != "" {
```

Comment:
(nitpick) a comment in Search.Section would be nice too - also, should we rename this in search.Section ?

Reply:
Thanks will do in a followup. As for renaming I can do it as well, it's fine, although we will still have places (client - api protocol) where old name needs to stay for compatibility.